src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -879,9 +879,25 @@ def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy=""
     # Get Helm chart path
     chart_path = utils.get_chart_path(registry_path, kube_config, kube_context, helm_client_location)
 
+    cmd_helm_values = [helm_client_location, "get", "values", "azure-arc", "--namespace", release_namespace]
+    if kube_config:
+        cmd_helm_values.extend(["--kubeconfig", kube_config])
+    if kube_context:
+        cmd_helm_values.extend(["--kube-context", kube_context])
+    
+    existing_user_values = open('userValues.txt', 'w+')
```

Comment:
Instead of using a file, we can directly use an object to store the output
https://github.com/Azure/azure-cli-extensions/blob/6da4c7bfc76e7ec37339ec3c864a1edc96f9476b/src/connectedk8s/azext_connectedk8s/custom.py#L1056

Reply:
Makes sense. Done